test/functional/feature_bip68_sequence.py

```diff
@@ -415,22 +402,19 @@ def activateCSV(self):
         min_activation_height = 432
         height = self.nodes[0].getblockcount()
         assert_greater_than(min_activation_height - height, 2)
-        self.generate(self.nodes[0], min_activation_height - height - 2, sync_fun=self.no_op)
+        self.generate(self.wallet, min_activation_height - height - 2, sync_fun=self.no_op)
         assert not softfork_active(self.nodes[0], 'csv')
-        self.generate(self.nodes[0], 1, sync_fun=self.no_op)
+        self.generate(self.wallet, 1, sync_fun=self.no_op)
         assert softfork_active(self.nodes[0], 'csv')
         self.sync_blocks()
 
     # Use self.nodes[1] to test that version 2 transactions are standard.
     def test_version2_relay(self):
-        inputs = [ ]
-        outputs = { self.nodes[1].getnewaddress() : 1.0 }
-        rawtx = self.nodes[1].createrawtransaction(inputs, outputs)
-        rawtxfund = self.nodes[1].fundrawtransaction(rawtx)['hex']
-        tx = tx_from_hex(rawtxfund)
+        mini_wallet = MiniWallet(self.nodes[1])
+        mini_wallet.rescan_utxos()
+        tx = mini_wallet.create_self_transfer()["tx"]
```

Comment:
nit in the last commit (can be fixed in a follow-up):

* MiniWallet shouldn't care about the node, so a single instance should suffice per test.
* Maybe the send_self_transfer function can accept a `version` option?

Reply:
May I create another PR to refactor those nits?

Reply:
Yes, if you agree with them and if the changes are possible (haven't checked myself)